pkg/storage/cache/dynamodb.go

```diff
@@ -85,18 +88,20 @@ func NewDynamoDBCache(parameters interface{}, log zlog.Logger) Cache {
 	if err != nil {
 		log.Error().Err(err).Msg("unable to load AWS SDK config for dynamodb")
 
-		return nil
+		return nil, err
```

Comment:
Maybe we can pass an invalid config in the environment for this test? But make sure it does not stay in the environment.